src/Apps/Artwork/Components/OtherAuctions.tsx

```diff
@@ -1,61 +1,27 @@
-import { Button, Flex, Serif } from "@artsy/palette"
-import { take } from "lodash"
-import React, { ReactNode } from "react"
-import { Col, Grid, Row } from "Styleguide/Elements/Grid"
-import { Media } from "Utils/Responsive"
+import { Box, Flex } from "@artsy/palette"
+import React from "react"
+import { data as sd } from "sharify"
+import { AuctionCardFragmentContainer as AuctionCard } from "Styleguide/Components/AuctionCard"
+import { Header } from "./OtherWorks/Header"
 
 interface OtherAuctionsProps {
-  children: ReactNode[]
+  auctions: any
```

Comment:
This can possibly be typed- `AuctionCard_sale[]` or something?

Reply:
Yea. Was not sure how to access it immediately. Will do another pass later or tomorrow. Or feel free to correct if you'll be touching this file.